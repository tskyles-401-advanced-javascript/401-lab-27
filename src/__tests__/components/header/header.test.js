import React from 'react';
import Header from '../../../components/header/header';

describe("<Header />", () => {
  it('is alive at app start', () => {
    let app = shallow(<Header/>);
    expect(app.find('header').exists()).toBeTruthy();
    expect(app.find('h1').exists()).toBeTruthy();
  });
  it('renders correctly', () => {
    const tree = renderer.create(<Header/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});