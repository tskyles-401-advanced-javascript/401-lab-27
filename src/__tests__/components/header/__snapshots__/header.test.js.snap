// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly 1`] = `
<header>
  <h1>
    Counters R Fun
  </h1>
</header>
`;
